RecoHGCal/TICL/plugins/TrackstersPCA.h

```diff
@@ -4,21 +4,50 @@
 #include "DataFormats/HGCalReco/interface/Trackster.h"
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include <vector>
+#include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
+
+typedef math::XYZVector Vector;
```

Comment:
This XYZVector is a double, is it needed?
If not we could use `ticl::Trackster::Vector`

Reply:
This typedef actually was not used anywhere